pages/ag/neuro-rse.md

```diff
@@ -0,0 +1,106 @@
+- [Alessandro Felder](github.com/alessandrofelder), Senior Research Software Engineer at the [Sainsbury Wellcome Centre's Neuroinformatics Unit](https://neuroinformatics.dev/), technical lead of the [BrainGlobe Initiative](https://brainglobe.info/)
+
+# Resources
+## Neuro-RSE groups
+
+- [NeuroRSE group](https://neurorse.flatironinstitute.org/) at Flatiron Center for Computational Neuroscience
+- [Neuroinformatics Unit](https://neuroinformatics.dev/) at the [Sainsbury Wellcome Centre](https://www.sainsburywellcome.org/web/)
+
+## Jobs
+
+- [US-RSE job board](https://us-rse.org/jobs/)
+- [RSE job board ](https://society-rse.org/careers/vacancies/) (international)
+- [Neuromatch job board](https://neuromatch.io/resources/job-board)
+- [SfN job board ](https://neurojobs.sfn.org/jobs/) (skews academic)
+- [Neurorumblr job board ](http://neurorumblr.com/) (skews academic)
+- [world wide neuro job board ](https://www.world-wide.org/jobs/) (skews academic)
+- [neurotechx job board](https://neurotechx.com/find-a-job/)
+- [Academic Data Science Alliance job board ](https://academicdatascience.org/jobs/) (skews academic)
+
+## Other places to find jobs
+
+- [jobRxiv job board](https://jobrxiv.org/) (skews academic)
+- [HigherEdJobs job board](https://www.higheredjobs.com/search/) (skews academic)
+- Institution/company websites
+- The usual job search websites (e.g., Google, Indeed, LinkedIn, Glassdoor)
+- Twitter (seriously)
+
+## Mailing lists
+
+- [comp-neuro mailing list](https://www.cnsorg.org/comp-neuro-mailing-list)
+- [cvnet mailing list](http://nephoscale.ewind.com/mailman/listinfo/cvnet) (skews academic)
+- [visionlist mailing list](http://visionscience.com/mailman/listinfo/visionlist_visionscience.com) (skews academic)
+- [connectionists mailing list](https://mailman.srv.cs.cmu.edu/mailman/listinfo/connectionists) (skews academic)
+
+## Google groups / mailing lists
+
+- [systems-neuroscience google group](https://groups.google.com/g/systems-neuroscience) (skews academic) 
+- [cosyne-discussions google group](https://groups.google.com/g/cosyne-discussions) 
+- [neural-ensemble google group](https://groups.google.com/g/neuralensemble)
+
+## Conferences
+- [Society for Neuroscience](https://www.sfn.org/) (largest American neuroscience conference)
+- [US-RSE](https://us-rse.org/events/conference/)
+- [COSYNE](https://www.cosyne.org/) (computational and systems neuroscience)
+- [Cognitive Computational Neuroscience](https://ccneuro.org)
+- [OHBM](https://humanbrainmapping.org/) (Organization for Human Brain Mapping)
+- [SciPy](https://conference.scipy.org/)
+- [INCF Neuroinformatics Assembly](https://www.incf.org/activities/incf-assembly)
+- [BRAIN Initiative Meeting](https://braininitiative.nih.gov/news-events/events/10th-annual-brain-initiative-conference)
+- [FENS](fens.org/) (Federation of European Neuroscience Societies, largest European neuroscience conference)
```

Comment:
Here and elsewhere (if agreed -- can sort them all) IMHO should be sorted alphabetically, e.g.
```suggestion
- [BRAIN Initiative Meeting](https://braininitiative.nih.gov/news-events/events/10th-annual-brain-initiative-conference)
- [Cognitive Computational Neuroscience](https://ccneuro.org)
- [COSYNE](https://www.cosyne.org/) (computational and systems neuroscience)
- [FENS](fens.org/) (Federation of European Neuroscience Societies, largest European neuroscience conference)
- [INCF Neuroinformatics Assembly](https://www.incf.org/activities/incf-assembly)
- [OHBM](https://humanbrainmapping.org/) (Organization for Human Brain Mapping)
- [SciPy](https://conference.scipy.org/)
- [Society for Neuroscience](https://www.sfn.org/) (largest American neuroscience conference)
- [US-RSE](https://us-rse.org/events/conference/)
```